cocos/2d/CCTMXLayer.h

```diff
@@ -294,6 +297,22 @@ class CC_DLL TMXLayer : public SpriteBatchNode
     */
     virtual std::string getDescription() const override;
 
+    /** Map from gid of animated tile to its instance.
+     *
+     * @return Map from gid of animated tile to its instance.
+     */
+    const std::map<uint32_t, std::vector<Vec2>>* getAnimTileCoord() {
+        return &_animTileCoord;
+    }
```

Comment:
Why not just return const reference like 
```c++
const std::map<uint32_t, std::vector<Vec2>>& getAnimTileCoord() const
```

Reply:
I tried before, but it will cause some problom about converting to lua values.

Reply:
Yep, it may have problem in lua.

Reply:
Like this:

`C:\projects\cocos2d-x\cocos\scripting\lua-bindings\auto\lua_cocos2dx_auto.cpp(101214): error C2770: invalid explicit template argument(s) for 'void object_to_luaval(lua_State *,const char *,T *)' [C:\projects\cocos2d-x\win32-build\engine\cocos\lua-bindings\luacocos2d.vcxproj]`